src/Pact/Native.hs

```diff
@@ -1338,9 +1342,12 @@ strToInt i as =
     doBase si base txt = case baseStrToInt base txt of
       Left e -> evalError' si (pretty e)
       Right n -> return (toTerm n)
-    doBase64 si txt = case parseB64UrlUnpaddedText' txt of
-      Left e -> evalError' si (pretty e)
-      Right bs -> return $ toTerm $ bsToInteger bs
+    doBase64 si txt = do
+      base64Behavior <- (\legacy -> if legacy then Legacy else Simplified) <$> isExecutionFlagSet FlagDisablePact49
```

Comment:
```suggestion
      base64Behavior <- (bool Legacy Simplified) <$> isExecutionFlagSet FlagDisablePact49
```

Reply:
Love it. I added this manually, with a comment.